internal/artifacts/module_arch.go

```diff
@@ -82,22 +83,24 @@ func ExecuteSoloBuild(source, target string, extensions []string, modulesNames [
 		for module := range selectedModulesMap {
 			err = collectSelectedModulesAndDependencies(mtaObj, selectedModulesWithDependenciesMap, module)
 			if err != nil {
-				return errors.Wrapf(err, getBuildErrMsg(buildFailedMsg, multiBuildFailedMsg, modulesNames))
+				return wrapBuildError(err, modulesNames)
 			}
 		}
 	} else {
 		selectedModulesWithDependenciesMap = selectedModulesMap
 	}
 
-	if len(selectedModulesWithDependenciesMap) > len(selectedModulesMap) {
-		logs.Logger.Infof(buildWithDependenciesMsg, "'"+strings.Join(sortModules(allModulesSorted, selectedModulesWithDependenciesMap), `',' `)+"'")
+	sortedModules := sortModules(allModulesSorted, selectedModulesWithDependenciesMap)
+
+	if allDependencies {
```

Comment:
Do you want to add `len(modulesNames) > 1` to this condition?

Reply:
added